### PR TITLE
[Connector-V2]Hive Source

### PR DESCRIPTION
**Pull request by CalvinKirs**

### CODE REVIEW

File: seatunnel-connectors-v2/connector-hive/src/main/java/org/apache/seatunnel/connectors/seatunnel/hive/source/HiveSource.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+import org.apache.seatunnel.shade.com.typesafe.config.Config;
+
+import com.google.auto.service.AutoService;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.util.List;
+
+@Slf4j
```
Comment (Hisoka-X) [inline seatunnel-connectors-v2/connector-hive/src/main/java/org/apache/seatunnel/connectors/seatunnel/hive/source/HiveSource.java]: You don't use `log` so why add `Slf4j` annotation?

File: tools/dependencies/known-dependencies.txt [hunk 2]
```diff
@@ -18,12 +19,16 @@ aggs-matrix-stats-client-7.5.1.jar
 aircompressor-0.10.jar
 airline-0.7.jar
 annotations-13.0.jar
+ant-1.6.5.jar
```
Comment (Hisoka-X) [inline tools/dependencies/known-dependencies.txt:22]: Do we need add LICENSE in seatunnel-dist?

Comment (CalvinKirs) [reply tools/dependencies/known-dependencies.txt:22]: yup, added.

File: seatunnel-connectors-v2/connector-hive/src/main/java/org/apache/seatunnel/connectors/seatunnel/hive/source/HiveSource.java (hunk 1 above)
Comment (CalvinKirs) [reply seatunnel-connectors-v2/connector-hive/src/main/java/org/apache/seatunnel/connectors/seatunnel/hive/source/HiveSource.java]: thanks, removed

**Review by Hisoka-X — APPROVED** [review]
LGTM

**Pull request merged**